pnpm-lock.yaml

```diff
@@ -1,5 +1,9 @@
 lockfileVersion: '6.0'
 
+settings:
+  autoInstallPeers: true
+  excludeLinksFromLockfile: false
```

Comment:
This was automatically added by pnpm. I dunno if this is a problem or how to avoid it.